### ISSUE TITLE
Test Figure 8 is giving ridiculous number of RPC count

### ISSUE THREAD

**Issue by junlong-gao** [junlong-gao/distributed_systems#20]
Current:
```
junlongg@MacBookPro-13 ~/d/2/s/raft> go test -run TestFigure82C
Test (2C): Figure 8 ...
labgob warning: Decoding into a non-default variable/field int may not work
  ... Passed --  32.1  5 30692   10
PASS
ok      raft    32.157s
```
Expected (https://pdos.csail.mit.edu/6.824/labs/lab-raft.html)
```
Test (2C): Figure 8 ...
  ... Passed --  33.2  5  852   53
```

**Comment by junlong-gao** [comment]
note `53` and `10`, only 10 committed?

**Comment by junlong-gao** [comment]
as it turns out, this is because test killing raft instance does not terminate the event loop. fixed in a36ffe836c01cc5f779b87b7c5cffb87a0976b40
